lib/linguist/languages.yml

```diff
@@ -34,6 +34,14 @@ ABAP:
   - .abap
   ace_mode: abap
 
+Apollo Guidance Computer:
```

Comment:
This entry should be right after the `Apex` entry (entries are sorted alphabetically with upper case characters first).
Sorry for the mess :s